starknet_py/tests/e2e/client/client_test.py

```diff
@@ -44,6 +46,22 @@ async def test_get_deploy_transaction(
         )
 
 
+@pytest.mark.asyncio
+async def test_get_declare_transaction(clients, declare_transaction_hash, class_hash):
+    # TODO extend this test to all clients
```

Comment:
Why can't we extend it now? I think that FullNodeClient has get_transaction method.

Reply:
At this moment, RPC `DeclareTransaction` is incompatible with devnet, since it does not expect `class_hash`.